src/main/java/org/folio/rest/impl/VoucherNumberImpl.java

```diff
@@ -3,49 +3,40 @@
 import io.vertx.core.AsyncResult;
 import io.vertx.core.Context;
 import io.vertx.core.Handler;
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
 import org.folio.rest.RestVerticle;
-import org.folio.rest.jaxrs.model.SequenceNumber;
 import org.folio.rest.jaxrs.resource.VoucherStorageVoucherNumber;
 import org.folio.rest.persist.PostgresClient;
-import org.folio.rest.tools.messages.MessageConsts;
 import org.folio.rest.tools.messages.Messages;
 import org.folio.rest.tools.utils.TenantTool;
 
 import javax.ws.rs.core.Response;
 import java.util.Map;
 
-import static io.vertx.core.Future.succeededFuture;
-import static org.folio.rest.jaxrs.resource.VoucherStorageVoucherNumber.GetVoucherStorageVoucherNumberResponse.respond500WithTextPlain;
-import static org.folio.rest.jaxrs.resource.VoucherStorageVoucherNumber.GetVoucherStorageVoucherNumberResponse.respond200WithApplicationJson;
-
 
 public class VoucherNumberImpl implements VoucherStorageVoucherNumber {
 
   private final Messages messages = Messages.getInstance();
 
-  private static final Logger log = LoggerFactory.getLogger(VoucherNumberImpl.class);
   private static final String VOUCHER_NUMBER_QUERY = "SELECT nextval('voucher_number')";
+  public static final String CURRENT_VOUCHER_NUMBER_QUERY = "SELECT currval('voucher_number')";
```

Comment:
Is this query returns sequence start value? According PostgreSQL https://www.postgresql.org/docs/8.2/functions-sequence.html `currval` function returns value most recently obtained with nextval for specified sequence but not sequence start value. Why you don't use query `SELECT start_value FROM <sequence name>;` from story description here?

Reply:
Ok. I tried that using `SELECT voucher_number.start_value FROM voucher_number` but for some reason that column does not exist and as a result of that I get this error : `Future{cause=ErrorMessage(fields=Map(Position -> 8, Line -> 3293, Hint -> Perhaps you meant to reference the column "voucher_number.last_value"., File -> parse_relation.c, SQLSTATE -> 42703, Routine -> errorMissingColumn, V -> ERROR, Message -> column voucher_number.start_value does not exist, Severity -> ERROR))}`
Am I missing anything?

Reply:
In any case start value is not current value. Probably, you can try to retrieve `minimum_value` instead `start_value` in your query  (https://www.postgresql.org/docs/9.1/infoschema-sequences.html).

Reply:
I changed and still getting similar error:
Query - `SELECT voucher_number.minimum_value FROM voucher_number`
`Future{cause=ErrorMessage(fields=Map(Position -> 8, Line -> 3293, File -> parse_relation.c, SQLSTATE -> 42703, Routine -> errorMissingColumn, V -> ERROR, Message -> column voucher_number.minimum_value does not exist, Severity -> ERROR))}`

Reply:
Actually, you will not able to retrieve start_value as well as another parameters because they are not presented for voucher_number sequence.
![image](https://user-images.githubusercontent.com/42964285/58054441-41f40280-7b63-11e9-810e-f03a14373277.png)
This is a question for investigation.

Reply:
I tried to recreate a new sequence and then query it and found that sequence itself is not creating `start_value` column which is strange
![image](https://user-images.githubusercontent.com/17807360/58145835-aa091e00-7c21-11e9-8b69-4d30f4465a10.png)

Reply:
@javalkar-varun ,
please, try query for your purpose
`SELECT start_value FROM pg_sequences WHERE sequencename = 'voucher_number'`
![image](https://user-images.githubusercontent.com/42964285/58186735-24877b80-7cbe-11e9-9e15-411a3c12b6d8.png)

Reply:
![image](https://user-images.githubusercontent.com/42964285/58187105-d7f07000-7cbe-11e9-92e7-8571b0302275.png)

Reply:
looks like pg_sequences is only supported as of postgres v10 (see https://www.postgresql.org/docs/10/view-pg-sequences.html).  I think the embededded postgres Varun was using is v9

Reply:
OK so we have a breaking change in postgres v10...

Reply:
Embedded Postgres version I'm using is `postgresql-10.3`. It is working now but I'm not sure if we should get the sequence from a view. What if `start_value` is moved from this view in future PostgreSQL version?

Reply:
Sounds like this is probably the best way to proceed.  If it turns out we need to also support earlier versions we'll handle that in separate stories